src/middleware/environment.js

```diff
@@ -131,5 +133,32 @@ module.exports = function environmentMiddleware(argv) {
 
     return resp
   }
+
+  // if there is no arapp.json and the command is not init default to the "global" config
+  // designed for the dao commands including dao acl
+  if (!module && !runsInCwd) {
```

Comment:
It could be nice to log that we're using the default configuration and where that's connecting to (perhaps under a verbose switch)

Reply:
Good idea! Added a better message as well for when the `environment` is not found.